library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -160,6 +160,10 @@ SAMLCore_3_2_1_b=The identifier for the version of SAML defined in this specific
 
 SAMLCore_3_2_1_c=The time value is encoded in UTC, as described in Section 1.3.3.
 
+SAMLCore_3_2_1_d=If a SAML responder deems a request to be invalid according to SAML syntax or \
```

Comment:
✏️ If we are passing in our own status code to test on, it may make sense to omit the last part of this reference:
`
If a SAML responder deems a request to be invalid according to SAML syntax or \
processing rules, then if it responds, it MUST return a SAML response message with a \
<StatusCode> element
`